### PR TITLE
Fix remove old username

### PR DESCRIPTION
**Pull request by ildyria**
Associated to PR : https://github.com/LycheeOrg/Lychee/pull/1403

### CODE REVIEW

File: scripts/main/lychee_locale.js [hunk 1]
```diff
@@ -332,8 +332,7 @@ lychee.locale = {
 	LOGIN_PASSWORD_CONFIRM: "Confirm Password",
 	LOGIN_CREATE: "Create Login",
 
-	PASSWORD_TITLE: "Enter your current username and password:",
-	USERNAME_CURRENT: "Current Username",
+	PASSWORD_TITLE: "Enter your current password:",
```
[the remark this answers is not in the document]
Comment (ildyria) [reply scripts/main/lychee_locale.js:335]: It is already changed. :D

[the remark this answers is not in the document]
Comment (ildyria) [reply scripts/main/lychee_locale.js:335]: Because it was already matching :D

[the remark this answers is not in the document]
Comment (ildyria) [reply scripts/main/lychee_locale.js:335]: np. :)

**Review by qwerty287 — APPROVED** [review]
Looks good then, didn't test.

**Pull request merged**